LINUX.md

```diff
@@ -0,0 +1,46 @@
+# Building on Linux
+Building on linux is made possible by cctools-port.
+
+Here is how, exactly, to install cctools and use it to compile.
+
+1. Dependencies
+- You need: clang (>=3.4) and everything Procursus normally needs. Get clang with apt.
+
+2. Get iOS SDK
+- You're going to need an iOS 13 SDK. Since you're on Linux, you're going to have to search online for one. I personally use https://github.com/xybp888/iOS-SDKs.
+
+3. Compress SDK
+- Compress iPhoneOS.sdk to iPhoneOS.sdk.tar.xz. On Ubuntu you can use the Files program and compress files by right-clicking and selecting Compress. 
+
+4. Get CCTools Port
+- You need Git for this. ```git clone https://github.com/tpoechtrager/cctools-port```
+
+5. Build iOS Toolchain
+- ```cd cctools-port/usage_examples/ios_toolchain && ./build.sh /PATH/TO/IPHONEOS-SDK/TAR/XZ arm64```
+
+6. Add to PATH
+- The last step made a toolchain in the "target" folder. We can use this to compile Procursus, but first we need to move it to our home folder and tell our system to use it.
+- ```mv target ~/cctools```
+- Then add this to your .profile:
+
+```
+if [ -d "$HOME/cctools" ] ; then
+    PATH="$HOME/cctools/bin:$PATH"
+fi
+```
+- And finally, ```source ~/.profile```. You'll need to re-source every time you use a new shell.
```

Comment:
`.profile` will get sourced by every new bash instance iirc, otherwise you could just put it in `.bashrc` or `.zshrc`

Reply:
Fixed